docs/en/05_Contributing/01_Code.md

```diff
@@ -77,15 +77,22 @@ there are any problems they will follow up with you, so please ensure they have
 ### Picking the right version
 
 The SilverStripe project follows the [Semantic Versioning](http://semver.org) (SemVer) specification for releases. 
-It clarifies what to expect from different releases, and also guides you in choosing the right branch
-to base your pull request on. In short, most pull requests should be based on the "default branch"
-which gets automatically chosen when creating a new pull request. This ensures your code
-gets into the next minor release of the current major release.
+It clarifies what to expect from different releases, and also guides you in choosing the right branch to base your pull request on.
+
+Most pull requests should be based on the "major branch" which gets automatically chosen when creating a new pull request.
```

Comment:
```suggestion
Most pull requests should be based on the "current major branch" which gets automatically chosen when creating a new pull request.
```

I think it's worth leaving in "default" too - as we should set the default branch to the "current major" in most cases.

Reply:
I'm thinking maybe it's worth removing this sentence altogether since it's quite situational and subjective. Does it actually bring much value to the documentation?

Reply:
I'm ambivalent. I feel like 95% of contributors don't actually read this anyway. It's probably better to have some guidance around how to choose your base branch rather than saying "most are this"

Reply:
rewrote this paragraph, please review